src/SolutionAcceleration.f90

```diff
@@ -0,0 +1,395 @@
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!                          Futility Development Group                          !
+!                             All rights reserved.                             !
+!                                                                              !
+! Futility is a jointly-maintained, open-source project between the University !
+! of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
+! can be found in LICENSE.txt in the head directory of this repository.        !
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!> @brief A Fortran 2003 module that encompasses a base class for solution
+!>        acceleration.  Module also includes both picard iteration and a
+!>        modified picard iteration.  Anderson Acceleration extends this base
+!>        class in the AndersonAcceleration module.
```

Comment:
Typically we have another line of `!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!` before the actual code starts...

Reply:
Done